pgcli/key_bindings.py

```diff
@@ -15,6 +15,8 @@ def pgcli_bindings(get_vi_mode_enabled, set_vi_mode_enabled):
     key_binding_manager = KeyBindingManager(
         enable_open_in_editor=True,
         enable_system_bindings=True,
+        enable_search=True,
```

Comment:
Yes it does. These bindings are now disabled by default, because it could be that the layout doesn't contain a user control that displays the search, and that would be very confusing.